README.md

```diff
@@ -77,7 +77,7 @@ output.
 With [npm](https://npmjs.org) do:
 
 ```
-npm install webworkify-webpack
+npm install webworkify-webpack --save-dev
```

Comment:
@amilajack please replace `--save-dev` with `--save` and I'll merge it because its a runtime dependency.

Thanks !